### ISSUE TITLE
Processing Library Download Link down

### ISSUE THREAD

**Issue by oysterboy** [kritzikratzi/edsdk-processing#3]
Hello!
It seems you're currently reworking your site,
so I can't download the edsdk-4.zip to try your library in processing.
Could you make it available somewhere else?
Thanks + Good luck with your new site!

**Comment by kritzikratzi** [comment]
that is correct. my server got wiped by accident, and i'm still in the dark if there is a backup or not. 
can you see if you can build the library yourself? (there are instruction in readme.md)

**Comment by oysterboy** [comment]
that's what i'm trying, tho its the first time i opened eclipse.
well, there's a first time for everything...
sorry to hear about your server..

**Comment by oysterboy** [comment]
Managed to compile the library.
Running HansiTest.java from eclipse
as well as the library examples from the Processing IDE
crashes JVM.
I'm Running Win 8.1 64bit and Processing 3.
While I'm still figuring out, if there might be  a 32bit/64bit mixup between referenced core.jar, 
DLLs and executed JVM.
Can this work at all? (Running 32bit versions of Processing/JVM/EDSDK Dlls)
Or do I need to b e on a 32 bit system?

**Comment by kritzikratzi** [comment]
you should have gotten both, 32 and 64 bit DLLs, with the EDSDK. 

you need to pass `-d32` as a jvm argument to start in 32 bit mode.

**Comment by oysterboy** [comment]
I sort of got it working with a 32bit jre7.
Going through the "Getting started" examples in edsdk4j,
I managed to change ISO,Shutter and Aperture settings on my Camera.
When it comes to downloading Liveview-Frames, or shooting images,
the JVM crashes.

**Comment by kritzikratzi** [comment]
ah yea, it's gonna get a bit rocky then. 
what's the crash? 

and what's your camera exactly? if it's new, you might have to regenerate the edsdk4j bindings.

**Comment by oysterboy** [comment]
it's a 450D, so nothing new here.
I get a Windows popup "JAVA SE binary funktioniert nicht mehr",
at times i get a direct crash with hs_err.log, see attachment.
That's a null pointer exception, but it might die trying to show me the windows popup,
if I look at the listed JAVA threads (shutdown,dispose etc.)?

Tomorrow I'll try and go with an older version of EDSDK Dlls, who knows....

[hs_err_pid7416.txt](https://github.com/kritzikratzi/edsdk-processing/files/299569/hs_err_pid7416.txt)

**Comment by oysterboy** [comment]
Just reverted back to Canon EDSDK v2.7.
=>  works!!!!!! Yay!
(Shouldn't have used the newest version thinking it must be better).
Tomorrow I'll test with a compiled library in Processing.
Was a nice getting-to-know-eclipse day today though...

**Comment by kritzikratzi** [comment]
cool :) 
glad you got it to working. i'm leaving this open until the website is back up.
